qa/tasks/cephfs/filesystem.py

```diff
@@ -611,6 +611,35 @@ def create(self):
 
         self.getinfo(refresh = True)
 
+    def destroy(self, reset_obj_attrs=True):
+        log.info('Destroying file system ' + self.name +  ' and related '
+                 'pools')
+
+        # make sure no MDSs are attached to given FS.
+        self.mon_manager.raw_cluster_cmd('fs', 'fail', self.name)
+        self.mon_manager.raw_cluster_cmd(
+            'fs', 'rm', self.name, '--yes-i-really-mean-it')
+
+        self.mon_manager.raw_cluster_cmd('osd', 'pool', 'rm',
+            self.get_metadata_pool_name(), self.get_metadata_pool_name(),
+            '--yes-i-really-really-mean-it')
+        for poolname in self.get_data_pool_names():
+            try:
+                self.mon_manager.raw_cluster_cmd('osd', 'pool', 'rm', poolname,
+                    poolname, '--yes-i-really-really-mean-it')
+            except CommandFailedError as e:
+                if e.exitstatus == 16: # EBUSY, this data pool is used
+                    pass               # by two metadata pools, let the 2nd
```

Comment:
I will need to dig through this PR page more thoroughly to find out the exact cause...

Reply:
I rewrote that code partially and this bit comes from previous version - https://github.com/rishabh-d-dave/ceph/blob/wip-djf-15070/qa/tasks/cephfs/filesystem.py#L603. 

I don't know if a data pool can be used by two meta pools at a time, but I left it as it is to avoid potential errors.

Reply:
The comment was not written by me. This entire block of code is present on master too - https://github.com/ceph/ceph/blame/master/qa/tasks/cephfs/filesystem.py#L365-L373. 

It was written in context to some tool for recovery data/metadata - https://github.com/ceph/ceph/commit/37bafff9f48bdd2e396aabc8a59c76b3be544e94. IMHO, we can leave it as it is.